splink/comparison_library.py

```diff
@@ -333,6 +329,74 @@ def _is_distance_subclass(self):
         return True
 
 
+class JaroAtThresholdsComparisonBase(DistanceFunctionAtThresholdsComparisonBase):
+    def __init__(
+        self,
+        col_name: str,
+        distance_threshold_or_thresholds: int | list = [0.9, 0.7],
+        include_exact_match_level=True,
+        term_frequency_adjustments=False,
+        m_probability_exact_match=None,
+        m_probability_or_probabilities_lev: float | list = None,
```

Comment:
Why is this called `m_probability_or_probabilities_lev`? I can see this appears in some of the other comparison functions.

I imagine it's a relic from the original levenshtein cl that never got renamed.

Reply:
I was unsure if it stood for "levenshtein" or "level". I think it makes sense to change for each comparator individually - will run through and update